config/samples/resources/dnsrecordset/dns-cname-record-set-with-dns-authorization-ref/certificatemanager_v1beta1_certificatemanagerdnsauthorization.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: certificatemanager.cnrm.cloud.google.com/v1beta1
+kind: CertificateManagerDNSAuthorization
+metadata:
+  name: certificatemanagerdnsauthorization-sample
```

Comment:
Nit: The name should be `dnsrecordset-dep-cname`. See naming convention here: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.Samples.md#directory-organization